dlt/cli/deploy_command.py

```diff
@@ -19,7 +19,7 @@
 from dlt.common.destination.reference import DestinationReference
 
 REQUIREMENTS_GITHUB_ACTION = "requirements_github_action.txt"
-DLT_DEPLOY_DOCS_URL = "https://dlthub.com/docs/walkthroughs/deploy-a-pipeline"
+DLT_DEPLOY_DOCS_URL = "https://dlthub.com/docs/walkthroughs/deploy-a-pipeline/deploy-with-github-actions"
```

Comment:
we should have a corresponding link in our code for composer. I'll take a look in the other PR (#401 )